rcsb/db/helpers/DocumentDefinitionHelper.py

```diff
@@ -220,6 +235,32 @@ def getSubCategoryAggregateFeatures(self, collectionName):
             logger.debug("Collection %s failing with %s", collectionName, str(e))
         return ret
 
+    def getIterableAttributeMetadata(self, collectionName, catName, atName):
+        """
+        Get metadata for iterable attributes (attribute types - scsv, csv, vbsv)
+        Example:
+
+            collection_iterable_attribute_metadata:
+                drugbank_core:
+                    - ATTRIBUTE_NAME: drugbank_info.drug_groups
+                    MIN_ITEMS: 1
+                    UNIQUE_ITEMS: TRUE
+        """
+        pD = {}
+        try:
+            # get atrribute metadata --
+            for clName, tDL in self.__cfgD["collection_iterable_attribute_metadata"].items():
+                if clName == collectionName:
```

Comment:
For this, was there a reason to not just get the `collectionName` from the dictionary directly? I think that would be more efficient than iterating over every key:value pair. E.g.,:

```python
tDL = self.__cfgD["collection_iterable_attribute_metadata"].get(collectionName, None)
if tDL:
    ...
```